main.rb

```diff
@@ -40,7 +19,7 @@ def h(text)
 end
 
 get '/memos' do
-  @memos = memos.sort_by { |memo| memo['created_at'] }
+  @memos = memo.all
```

Comment:
```suggestion
  @memos = Memo.all
```

インスタンスにしてもあまり意味がない（アプリ内で1回しかnewされない）ので、Memoクラスのメソッドは全部クラスメソッドにした方がいいかもしれません。